### ISSUE TITLE
Release 0.4.0

### ISSUE THREAD

**Issue by lulf** [voteblake/csaf-rs#3]
Hi! I was happy to find this crate and wanted to use it in my project. Are there any 'blockers' for releasing the 0.4.0 version of this crate? I need some of the functionality that has been added since 0.3.2

**Comment by voteblake** [comment]
Hi @lulf - short answer is our 10 month old baby 🤣. It hasn’t been a top priority for any work, so it fell down the list. I can spend some time this afternoon to come back up to speed and get at least the work that’s already been done published as a new version. I expect I’m behind the current draft standard by at least a bit again. Thanks for reporting issues as you find them.

**Comment by lulf** [comment]
I can totally relate to that, I'm not in a hurry so take your time :) I made one PR (#4, not sure if its the correct fix), and relying on git dependency on my branch for now.

**Comment by voteblake** [comment]
Thanks for the poke on this @lulf! I dug up some previously uncommitted WIP which was blocked on a change to the CPE crate that shipped in September. I incorporated that, your contribution, and a couple other maintenance upgrades in a v0.4.0 release which is live on crates.io. I suspect we're now a bit behind the current editor draft of the spec, which I might have some time to investigate this weekend. I also opened a couple other issues to track improvements that have been on my mind.
